openSRS-App/openSRS
# Completed AddCard functionality onto Card Component

## Description:
- Added AddCard functionality to application to retrieve user input for front and back cards to post into database

## Changes I Made:
- Implemented onPress functionality to Add Card button in Card Component 
- Added AddCard component for an input field to post a card into the database

## How to Test:
- Navigate through App to Card Component, select Add a Card, and submit a form. A console.log will appear in the inspector console of the successfully added card.
